### DISCUSSION TITLE
PQ Splitter does not close the windows by itself

### DISCUSSION THREAD

**Discussion by capoaira** [2Abendsegler/GClh#2230] [Questions]
Moin,
ich nutzte den PQ Splitter eigentlich nicht, aber mir ist aber eben beim Testen aufgefallen, dass nur 5 PQ auf einmal erzeugt werden können und sich die Fenster nicht automatisch schließen. Damit es weitergeht, musste ich die Fenster manuell schließen, daher meine Frage: Gibt es dafür einen Grund, etwas was ich übersehen habe, im Browser einstellen muss, oder ist es ein Bug?

**Comment by 2Abendsegler** [comment]
Ich habe das gerade getestet. 6 PQs wurden automatisch angelegt und die Fenster wurden automatisch geschlossen.
Firefox aktuell.
Mit welchem Browser arbeitest du.

**Reply by capoaira** [reply]
Ebenfalls aktueller Firefox, Pop-Up Blocker ist natürlich deaktiviert...

**Comment by 2Abendsegler** [comment]
Ich hatte noch mit Sprache Deutsch bei Project-GC getestet, auch ok. Und ich habe beide Tabellen mit 1000 und 500 getestet, auch ok.
Da habe ich jetzt auch keine Idee mehr. 
Ich konnte auch die Stelle, wo wir die Popups schließen, gerade nicht finden. Vielleicht wäre mir dann noch etwas eingefallen.
Vielleicht kannst du ein Issue aufmachen und mal danach sehen, sobald sich die Motivation einstellt.
